support-frontend/assets/components/legal/contribLegal/contribLegal.jsx

```diff
@@ -19,7 +23,8 @@ type PropTypes = {|
 // ----- Component ----- //
 
 export default function ContribLegal(props: PropTypes) {
-
+  const contactUs = useDotcomContactPage() ? <ContactPageLink />
+    : <a href={contributionsEmail[props.countryGroupId]}>contact us here</a>;
```

Comment:
nitpick and maybe not really our decision, but this is the one place we use `contact us here` instead of just `contact us`

Reply:
Yeah that is true but I put it down to being more of an inconsistency than a deliberate decision. Happy to change if you think it's worth it?

Reply:
I agree that it's probably not deliberate, I would change it